src/utility/svd.jl

```diff
@@ -389,3 +411,177 @@ function ChainRulesCore.rrule(
 
     return (U, S, V, info), tsvd!_itersvd_pullback
 end
+
+# scalar inverses with a cutoff tolerance and Lorentzian broadening
+function _safe_inv(x, tol, ε=0)
+    if abs(x) < tol
+        return zero(x)
+    else
+        return iszero(ε) ? inv(x) : _lorentz_broaden(x, ε)
+    end
+end
+
+# Lorentzian broadening for divergent term in SVD rrule, see
+# https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.7.013237
+function _lorentz_broaden(x, ε=1e-13)
```

Comment:
```suggestion
function _lorentz_broaden(x, ε=eps(real(scalartype(x)))^(3/4))
```
The power is a bit arbitrary, but at least this should be better behaved for non 64bits types